### ISSUE TITLE
Add toggle button for viewing older Ordnance Survey MasterMap releases

### ISSUE THREAD

**Issue by edwardchalstrey1** [colouring-cities/colouring-london#107]
Explore potential of retention and visualisation of 2016 and subsequent annual OSMM releases with OS
Can we click back through older Ordnance Survey MasterMap layers?

- [ ] colouring-cities/colouring-core#818

**Comment by polly64** [comment]
@mdsimpson42 do we have a toggle button?

**Comment by mdsimpson42** [comment]
I don't think we have a toggle button. I'm not sure how the mastermap versions are managed/updated...
